public/index.html

```diff
@@ -5,20 +5,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="icon" type="image/png" href="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2016/05/cc-site-icon-300x300.png" sizes="192x192">
+    <link rel="apple-touch-icon-precomposed" href="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2016/05/cc-site-icon-300x300.png">
     <meta name="twitter:card" content="summary" />
     <meta name="twitter:site" content="@creativecommons" />
     <meta name="twitter:creator" content="@creativecommons" />
-    <meta property="og:url" content="https://creativecommons.org/choose/" />
-    <meta property="og:title" content="Creative Commons License Chooser" />
-    <meta property="og:description" content="This chooser helps you determine which Creative Commons License is right for you in a few easy steps." />
-    <meta property="og:image" content="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2019/11/WP-Plugin-Screenshot-440x250.png" />
+    <meta property="og:url" content="https://beta-chooser.creativecommons.org" />
+    <meta property="og:title" content="Choose a License" />
+    <meta property="og:type" content="website">
+    <meta property="og:description" content="Want to license your work with Creative Commons, but not sure where to start, or which license is right for you? Use our license chooser!" />
+    <meta property="og:image" content="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2016/05/cc-site-icon-300x300.png" />
```

Comment:
I did say "Creative Commons logo", but I think this one would be better: https://mirrors.creativecommons.org/presskit/logos/cc.logo.large.png